views/partials/footer.njk

```diff
@@ -1,6 +1,10 @@
 {% from "govuk/components/footer/macro.njk" import govukFooter %}
 
-{% include "../components/feedbackBanner/feedbackBanner.njk" %}
+<div style="background: {{ feedbackBannerBackground or '#fff' }}" class="govuk-!-display-none-print">
+  <div class="govuk-width-container">
+    {% include "../components/feedbackBanner/feedbackBanner.njk" %}
+  </div>
+</div>
```

Comment:
maybe it would be nicer to make the banner a macro and parameterise it. 

Would probably be better to change the class rather than have in-line styles

Reply:
Now sorted in last commit